start-test-solr.sh

```diff
@@ -1,11 +1,13 @@
-#!/bin/sh
+#!/bin/bash
 
 set -e
 
 SOLR_VERSION=4.7.2
 
-export SOLR_ARCHIVE="${SOLR_VERSION}.tgz"
+export SOLR_ARCHIVE="solr-${SOLR_VERSION}.tgz"
+export SOLR_DIR="solr-${SOLR_VERSION}"
```

Comment:
Good idea, it would be much cleaner that way. And thanks for responding so quickly!